page-data.json

```diff
@@ -7,5 +7,6 @@
   "imageHeight": "765",
   "facebookPage": "bankempatii",
   "twitterPage": "BankEmpatii",
-  "twitterCreator": "tomekmagulski"
+  "twitterCreator": "tomekmagulski",
```

Comment:
ja bym zostawił bankEmpatii - tam to chyba ma zastosowanie, jak autorów jest więcej i oni chcą być rozróżnialni - a tutaj moja skromna osoba nie jest potrzebna.